README.md

```diff
@@ -43,7 +43,11 @@ Node.js >= 6.0.0 required.
 - [Plugins](https://github.com/search?q=topic%3Aegg-plugin&type=Repositories)
 - [Frameworks](https://github.com/search?q=topic%3Aegg-framework&type=Repositories)
 
-> English docs is still working in process, see [#360](https://github.com/eggjs/egg/issues/360).
+> All the source, comment and README is english.
+>
+> Only guide docs is still working in process, see [#363](https://github.com/eggjs/egg/issues/363).
```

Comment:
The document is still [working in process](https://github.com/eggjs/egg/issues/363),